src/featureFlags.js

```diff
@@ -57,7 +58,10 @@ export class FeatureFlags extends events.EventEmitter {
       if (shouldIgnore) {
         return false;
       }
-      this.client.updateContext({ userId: this.userId });
+      this.client.updateContext({
+        userId: this.userId,
+        stage: this.network,
```

Comment:
Shouldn't is be:

```
        properties: {
          stage: this.network,
        },
```
?

And do we have different feature flags for mac/windows/linux?

Reply:
You are absolutely correct.

Done in 521280f49f007a9aff049ebcef0019ea9b43300d

Also sending the current OS